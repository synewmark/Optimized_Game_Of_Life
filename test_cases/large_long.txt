version https://git-lfs.github.com/spec/v1
oid sha256:f286a6c1b091dc87df69f8f1dfc488feeff710042716fa31dae45a46ab0ac72c
size 200040022
