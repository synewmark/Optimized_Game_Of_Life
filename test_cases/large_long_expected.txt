version https://git-lfs.github.com/spec/v1
oid sha256:0a54510f4c0b845d76eca4f8365af813872f7208508b05e6ba1f2541d0a6d22c
size 200020000
